CMakeLists.txt

```diff
@@ -1,15 +1,14 @@
 # use cmake 3.12 or later
 cmake_minimum_required (VERSION 3.12)
 
-project(vlneval VERSION 0.0.1 LANGUAGES CXX)
+set(PROJ_VERSION 0.0.1)
 
-# if the CETBUILDTOOLS_VERSION environmental variable is defined, use cetbuildtools
-if(DEFINED ENV{CETBUILDTOOLS_VERSION})
-# using cetbuildtools to create a ups product
+# if the CETMODULES_VERSION environmental variable is defined, use cetmodules
+if(DEFINED ENV{CETMODULES_VERSION})
 
-    # cetbuildtools contains our cmake modules
-    # Note that find package will add cetbuildtools Modules to CMAKE_MODULE_PATH
-    find_package(cetbuildtools REQUIRED)
+    # when using cetmodules, find_package(cetmodules..) must come before project(...)
+    find_package(cetmodules 2.29.06 REQUIRED)
+    project(vlneval VERSION ${PROJ_VERSION} LANGUAGES CXX)
```

Comment:
The `vlneval/ups/setup_for_development -p e20` fails on this line with the following message:
```
ERROR:
ERROR: VERSION as specified in vlneval/CMakeLists.txt:project() (${PROJ_VERSION}) has an
ERROR: impermissible non-numeric component "${PROJ_VERSION}": remove from project()
ERROR: and set ${PROJECT_NAME}_CMAKE_PROJECT_VERSION_STRING to ${PROJ_VERSION}
ERROR: before calling cet_cmake_env()
ERROR:
ERROR: generation of setup script has failed
```
Should I revert the `${PROJ_VERSION}` to simply `0.0.1`?

Reply:
This works in our tests.  Did you change the cetmodules version in both places?  
 The intent of providing ${PROJ_VERSION} was simply to put the version number in a single place so there was no confusion when building with or without cetmodules.

Reply:
I was wrong.  See the new commit.

Reply:
Note: `setup_for_development` requires a literal version here so we can detect and support non-numeric version components as required by many of our users. It also enables us to provide tools such as `mrb uv` and `tools/migrate` to assist development and migration, respectively.